e2e/tests/init_test.go

```diff
@@ -100,13 +106,13 @@ func (s *suite) TheUserRunsTheCommand(cmd string) {
 	res, err := s.pocketd.RunCommand(cmds...)
 	s.pocketd.result = res
 	if err != nil {
-		s.Fatalf("error running command %s: %s", cmd, err)
+		s.Fatalf("error running command %s: %s", cmd, err) //nolint:typecheck // This method does exist
```

Comment:
Do you know what the issue with `s.Fatalf` is?

Reply:
Its embedded in the struct `s` as a gocuke.T type and the linter doesn't detect this - and `s` inherits this method from the embedded field